tests/integration/test_zeromq.py

```diff
@@ -0,0 +1,160 @@
+from threading import Event
+
+from compas_eve import Message
+from compas_eve import Publisher
+from compas_eve import Subscriber
+from compas_eve import Topic
+from compas_eve import set_default_transport
+
+try:
+    from compas_eve import ZeroMQTransport
+    ZEROMQ_AVAILABLE = True
+except ImportError:
+    ZEROMQ_AVAILABLE = False
```

Comment:
Remove conditional logic from tests, always test zeromq. 
Also, ZeroMQTransport should be importable from `compas_eve.zeromq` instead of from `compas_eve`